less/about-me.less

```diff
@@ -1,3 +1,8 @@
+body {
+    background-image: url("https://files.slack.com/files-pri/T4JUEB3ME-FPMRT6D2L/courtbg.jpg");
+    background-size: 2000px;
+}
+
 header {
     margin-top: 40px;
     margin-bottom: 40px;
```

Comment:
Looks really good...jealous of how DRY it is.  Was just going to say add in a mixin & its perfect.